src/components/Tokens/TokenDetails/TokenDetail.tsx

```diff
@@ -174,9 +175,15 @@ export function AboutSection({ address, tokenDetailData }: { address: string; to
   )
 }
 
-export default function LoadedTokenDetail({ address }: { address: string }) {
+export default function LoadedTokenDetail({
+  address,
+  connectedChainId,
+}: {
+  address: string
+  connectedChainId: number | undefined
+}) {
   const token = useToken(address)
-  const currency = useCurrency(address)
+  let currency = useCurrency(address)
```

Comment:
do you know if we intend to keep useCurrency? seems hard to follow

Reply:
I think the point is to get the appropriate logo

Reply:
Oh wild. Sounds good